mscore/musescore.cpp

```diff
@@ -7832,22 +7845,29 @@ void MuseScore::init(QStringList& argv)
             qApp->processEvents();
             }
 
-      if (!MScore::noGui)
+      // Best not to show this since the font used to display the message
+      // isn't updated till updateUiStyleAndTheme()
+      // showSplashMessage(sc, tr("Updating interface style and theme…"));
```

Comment:
Maybe better "... user interface..."? Doesn't matter too much, being comment out, but...

Reply:
you could still show it after the fact.

Reply:
and maybe add a short delay? Less than a second for sure, maybe some 200ms?

Reply:
in the `showSplashMessage()`

Reply:
It's not useful to show it after the operation is done.

Reply:
Well, if you can't show before it might be useful to show after. Better late than never?

Reply:
It doesn't make sense to put a delay there just to "show it". If it's done it's done, and if something goes wrong afterwards it won't be wrongly blamed.